src/DSE.Open.Results/ResultBuilder.cs

```diff
@@ -9,12 +9,19 @@ namespace DSE.Open.Results;
 public abstract class ResultBuilder<TResult>
     where TResult : Result
 {
-    public ICollection<Notification> Notifications { get; } = new List<Notification>();
+    public virtual ResultStatus Status { get; set; }
```

Comment:
Why virtual?

Reply:
Was wondering if collection result builder might override with respect to `Found` if collection empty.